### DISCUSSION TITLE
建议：是否能够将生词本上的单词标记在网页上，当鼠标停留到标记上x秒弹出翻译窗口。

### DISCUSSION THREAD

**Discussion by Maverick3a1** [crimx/ext-saladict#1500] [Ideas]
@crimx 使用生词本有感，发现单词虽然记下来了，但是只是完成了收集的第一步，跟一个完整的学习流程没有衔接上，如果插件能够实现在日常阅读过程中的上下文场景下进行思考理解，那对于英语学习应该是很有帮助的。

**Comment by crimx** [comment]
感谢反馈！对上下文进行思考理解也是我最想要的功能哈哈，但目前的技术很难做到这个效果，最多只能标记一下出现多少次这样。

**Reply by Maverick3a1** [reply]
我的设想是，把网页里面存在于生词本中的词汇进行高亮或者下划线，当鼠标移到标记或点击标记就显示该词汇的查词窗口，有了上下文的信息补充，此功能将有助于加强在实际阅读环境下对生词的训练，以此形成对词汇用法跟使用场景的深入理解。
这会是个令人激动的功能，有点迫不及待了。

**Comment by ElonMuMu** [comment]
对该功能同样期待, 可以参考下Burn Vocabulary这个插件.
(这个插件劝退我的原因是只能用bing.com.cn的翻译, 装了这个插件, New bing会自动重定向回到中国)

**Reply by ElonMuMu** [reply]
评论补充, 另一个实现该功能的插件是Relingo.
推荐度: Relingo>Burn Vocabulary
